functions/afaf/afaf.js

```diff
@@ -0,0 +1,25 @@
+const { discordMessageEmbed } = require('../config');
+const { blaccLogo } = require('../constants');
+
+exports.askAnonymously = (question, preferredChannelId, discordClient) => {
+	discordClient.channels.cache
+		.get(preferredChannelId)
+		.send(embedMessage(question));
+};
+exports.getChannel = async (author, currChannel, serverChannels) => {
```

Comment:
Let's rename this to something like "getPreferredChannel" to be more specific on what this does.